hera_cal/tests/test_abscal.py

```diff
@@ -531,13 +555,13 @@ def test_omni_abscal_arg_parser(self):
         a = hc.abscal.omni_abscal_arg_parser()
 
     def test_abscal_run(self):
```

Comment:
did you add an `abscal_run` call with `phs_slope_cal = True`

Reply:
Done.